### PR TITLE
Implement PanelPreferencesToolbar component

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium-oss/react-science/issues/520

### CODE REVIEW

**Comment by lpatiny** [comment]
I think the buttons should be closer to each other like in NMRium

image

**Comment by wadjih-bencheikh18** [comment]
Nmrium:
image

Our component:
image

The two are size `16px`

File: src/components/toolbar/PanelPreferencesToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { ReactNode } from 'react';
+import { FaCheck, FaTimes } from 'react-icons/fa';
+
+import { Button } from '../button/Button';
+
+interface PanelPreferencesToolbarProps {
+  children?: ReactNode;
```
Comment (targos) [inline src/components/toolbar/PanelPreferencesToolbar.tsx:9]: The toolbar should not accept children, and not render the entire panel container. Just the toolbar, like in your other PR.

Comment (wadjih-bencheikh18) [reply src/components/toolbar/PanelPreferencesToolbar.tsx]: okey, fixed

**Pull request merged**